js/index.js

```diff
@@ -1,6 +1,60 @@
-// Iteration 1: Names and Input
+//console.log("I´m ready!")
 
-// Iteration 2: Conditionals
+let hacker1 = "Ana"
+  console.log(`The driver's name is ${hacker1}`)
 
+let hacker2 = "Tobias";
+    console.log(`The navigator's name is ${hacker2}`)
 
-// Iteration 3: Loops
+if (hacker1.length === hacker2.length){
+  console.log(`Wow, you both have equally long names, ${hacker1.length} characters!`)
+}
+else if (hacker1.length < hacker2.length){
+  console.log(`It seems that the navigator has the longest name, it has ${hacker2.length} characters.`)
+
+}
+else if (hacker1.length > hacker2.length){
+  console.log(`The driver has the longest name, it has ${hacker1.length}characters.`)
+}
+
```

Comment:
good job, but here you could also do just if, else if, else (instead of if, else if, else if)